sdk/core/aborter/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@azure/core-aborter",
+  "version": "1.0.0",
+  "description": "Microsoft Azure SDK for JavaScript - Aborter",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/aborter.js",
+  "browser": {
+    "./dist/index.js": "./browser/index.js"
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:es6": "tsc -p tsconfig.json",
+    "build:nodebrowser": "rollup -c 2>&1",
+    "build:test": "rollup -c rollup.test.config.js 2>&1",
+    "build": "npm run build:es6 && npm run build:nodebrowser",
+    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm dist-test types temp browser/*.js* browser/*.zip statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
```

Comment:
`../../.prettierrc.json`

Reply:
Updated location in `check-format` and `format`. Thanks!